src/commonMain/kotlin/exchange.dydx.abacus/utils/AnalyticsUtils.kt

```diff
@@ -215,8 +215,8 @@ class AnalyticsUtils {
         return iMapOf(
             "orderId" to order.id,
             "clientId" to order.clientId,
-            "type" to order.type.rawValue,
-            "side" to order.side.rawValue,
+            "type" to order.type?.rawValue,
+            "side" to order.side?.rawValue,
             "status" to order.status.rawValue,
             "timeInForce" to order.timeInForce?.rawValue,
             "marketId" to order.marketId,
```

Comment:
hmm why don't we have to update `marketId`, `size`, `price` here as well?

Reply:
since they'll just be undefined and we don't need to do a `?.` here!